web/templates/generated/incuna-surveys/form/base/proportion-field.html

```diff
@@ -11,16 +11,19 @@ <h4 class="title" ng-bind="title"></h4>
                         {{ form[id].$invalid ? 'has-error' : '' }}"
                         >
                     
-                        AMOUNT ALLOCATED (use for css): {{ total - model[$index] }}
                         <div class="proportion-input" integer-field model="model[$index]" id="field.id" form="form"></div>
-                        <div
-                            aif-slider-input
-                            model="model[$index]"
-                            ceiling="100"
-                            slider-low-label="0%"
-                            slider-high-label="100%"
-                        ></div>
-                    
+                        <div class="proportion-slider">
+                            <div
+                                clearable-rz-slider
+                                model="model[$index]"
+                                rz-slider-options="optionsPerSlider[$index]"
+                                slider-low-label="0%"
+                                slider-high-label="100%"
+                                ceiling="100"
```

Comment:
Should `ceiling` be specified here, as well as the `ceil` in `optionsPerSlider`?